pyramid_swagger/exceptions.py

```diff
@@ -2,10 +2,10 @@
 import sys
 
 import six
-from pyramid.httpexceptions import HTTPClientError, HTTPInternalServerError
+from pyramid.httpexceptions import HTTPBadRequest, HTTPInternalServerError
 
 
-class RequestValidationError(HTTPClientError):
+class RequestValidationError(HTTPBadRequest):
```

Comment:
I'm not sure I understand the difference between inheriting from HTTPBadRequest vs HTTPClientError. http://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html makes it look like maybe the former isn't implying an explicit return code? Mind helping me out here?